ONBOARDING.md

```diff
@@ -42,4 +43,36 @@ Add the new plugin to the [opensearch-plugins meta](https://github.com/opensearc
 
 3. For backward compatibility testing, the `test-workflow` runs backward compatibility tests available in the plugin repository, (see [reference]((https://github.com/opensearch-project/anomaly-detection/blob/d9a122d05282f7efc1e24c61d64f18dec0fd47af/build.gradle#L428))). Like integration test, it has a set of configurable options defined in opensearch-1.3.0-test.yml, [example](manifests/1.3.0/opensearch-1.3.0-test.yml).
 
-    1. It supports two test configs - `with-security` and `without-security`, which runs test with security plugin enabled and disabled respectively. Choose one or both depending on what your plugin integration tests support.
\ No newline at end of file
+    1. It supports two test configs - `with-security` and `without-security`, which runs test with security plugin enabled and disabled respectively. Choose one or both depending on what your plugin integration tests support.
+
+
+## Onboarding to universal / 1-click release process:
+
+This document describes steps to onboard a new component to universal or 1-click release process.
+
+See https://github.com/opensearch-project/opensearch-build/issues/1234 for details about end to end workflow.
+
+1. Please ensure that [opensearch-ci-bot](https://github.com/opensearch-ci-bot) has the write access to your repository. 
```

Comment:
Add sentence on if the bot does not exist, contact OSEE team.

Reply:
Team are being removed. Will add maintainer of this repo. Thanks!

Reply:
Or link code owner file is better?